konfera/forms.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+from konfera.models import Speaker, Talk
+
+
+class SpeakerForm(forms.ModelForm):
+    class Meta:
+        model = Speaker
+        fields = '__all__'
+
+
+class TalkForm(forms.ModelForm):
+
+    class Meta:
+        model = Talk
+        exclude = ['status', 'primary_speaker', 'secondary_speaker']
```

Comment:
exclude event, and update url and view that cfp is part of event and value is predefined from url. This is better UX for user, he has one less field on the form and in case there are multiple evnets user will have the right one.